catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/EntityRepository.java

```diff
@@ -524,6 +584,39 @@ private void recoverDeletedRelationships(T original) {
     }
   }
 
+  public enum Include {
```

Comment:
This could be defined in JSON basic.json and generate the code? What do you think?

Reply:
I'm on it.

Reply:
Should I put it in the folder `api` or in the folder `type`? I cannot put it in basic.json definitions because it looks that our tool doesn't generate definition unless they are used in another object. I would put it in `api` because is something we have only in the REST API. WDYT?

Reply:
I'll put it in type, it seems more consistent with what we have done so far.